Ticket-booking-spring-app/src/main/java/com/example/demo/exception/RoomException.java

```diff
@@ -0,0 +1,15 @@
+package com.example.demo.exception;
+
+public class RoomException extends RuntimeException {
```

Comment:
Ta nazwa wyjątku słabo przekazuje czego on dotyczy. Wcześniejsza - `InvalidReservation` już jest lepsza. Tutaj kompletnie nie wiadomo co poszło nie tak, poza tym że dotyczy to sali :)

Reply:
To prawda, w tym wypadku chodziło mi akurat o RoomNotFound - to lepsza nazwa :)